src/components/newComponent/pos/ProductSearch.tsx

```
@@ -0,0 +1,145 @@
+          <div className="text-center py-8">
+            <Package className="w-8 h-8 text-gray-400 mx-auto mb-2" />
+            <p className="text-gray-500">Aucun produit trouvé</p>
+          </div>
+        ) : (
+          filteredProducts.map((product) => (
+            <div
+              key={product.id}
+              onClick={() => onProductSelect(product)}
+              className="flex items-center justify-between p-3 border rounded-lg hover:bg-gray-50 cursor-pointer transition-colors"
+            >
+              <div className="flex-1">
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)